app/api/roles.ts

```diff
@@ -12,8 +12,7 @@
  * it belongs in the API proper.
  */
 import { useMemo } from 'react'
-
-import { lowestBy, sortBy } from '~/util/array'
+import * as R from 'remeda'
```

Comment:
It turns out this tree-shakes perfectly well, so we don't need to bother with any manual `import { sortBy } from 'remeda'`.